src/components/WebexVideoSettings/WebexVideoSettings.scss

```diff
@@ -1,4 +1,12 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-video-settings {
+  h5 {
```

Comment:
this looks very similar to the Participants title (only difference is that that one is bold, this one should be medium according to figma).

To avoid duplication, can you create a simple generic component named Title to be used as such:
```jsx
<Title>Speaker</Title>
```

we could then use it for participants as well (in a different PR):
```jsx
<Title bold>Participants ({count})</Title>
```